tests/.pylintrc

```diff
@@ -44,7 +44,14 @@ disable=
   import-outside-toplevel,
   missing-class-docstring,
   missing-function-docstring,
-  no-self-use
+  no-self-use,
+  no-member, # because of qnode decorator
+  comparison-with-callable,
+  unsubscriptable-object, # because of qnode decorator
+  not-callable, # because of qnode decorator
```

Comment:
I wonder if we could improve the qnode somehow so that pylint can figure out these things.

Reply:
opened June 10, 2014. guess its a hard problem.